docs/conf.py

```diff
@@ -74,6 +127,47 @@
     '.md': 'markdown',
 }
 
+autoapi_type = "python"
+autoapi_dirs = [f"{_base_path}/lib/sedna"]
+autoapi_options = [
+    'members', 'undoc-members', 'show-inheritance',
+    'show-module-summary', 'special-members', 'imported-members'
+]
+
+extlinks = {
+    "issue": f"{BASE_URL}issues",
+    "pr": f"{BASE_URL}pull"
+}
+
+
+# hack to replace file link to html link
+def ultimateReplace(app, docname, source):
+    path = app.env.doc2path(docname)
+    _match = re.compile("\(/([^/]+)/([^)]+)\)")
+    if path.endswith('.md'):
+        new_line = []
+        for line in source[0].split('\n'):
+            error_link = _match.search(line)
+            if error_link:
+                _path, _detail = error_link.groups()
+                if _path in ("docs", "examples") and ".md" in _detail.lower():
```

Comment:
Add some comments to help better understand these links mangling

Reply:
check